setup.py

```diff
@@ -12,11 +15,34 @@ def validate_pywin32():
         import win32gui  # NOQA
         import win32con  # NOQA
     except ImportError as e:
-        print e
+        print(e)
         sys.exit("Please make sure you already install the pywin32 properly. You can download the latest version from here https://sourceforge.net/projects/pywin32/")
 
+#
+# Clean up some critical files which will crash the program
+#
+clean_file_list = [
+    os.path.join('ejenti', 'ejenti.pyc'),
+]
+
+for clean_file in clean_file_list:
+    print('Checking {} ... '.format(clean_file), end='')
+    if os.path.exists(clean_file):
+        print('found')
```

Comment:
I think `print("Found" + clean_file)` or `print('Found {} ... '.format(clean_file), end='')` could be better, and it should be the same for L39.

Reply:
`print("Found" + clean_file)` will become
```bash
Checking FOO ... Found FOO
```

`print('Found {} ... '.format(clean_file), end='')` will become
```bash
Checking FOO ... Found FOO ... 
```

Reply:
Sorry, I didn't read it carefully. :P